src/data_utils.py

```diff
@@ -57,28 +108,32 @@ def __post_init__(self):
             ), "You must specify `data_dir` when `from_tfrecords=False`."
 
 
-class LibriSpeechDataLoader:
+@dataclass
+class TimitDataLoaderArgs:
+    data_dir: str = "../data/timit/data/TRAIN"
```

Comment:
Prefer loading this path from a configuration file.

Reply:
@sayakpaul, this is the `Args` object already. I didn't understand the need of putting this path in separate config file as this object will be called & updated as per need in other script.

something like this: `args = TimitDataLoaderArgs(data_dir="something")`

If I am understanding your comment wrongly, can you explain it again?

Reply:
Makes sense. I was a bit confused earlier.